src/libs/Navigation/linkingConfig/config.ts

```diff
@@ -0,0 +1,515 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type {LinkingOptions} from '@react-navigation/native';
+import type {RootStackParamList} from '@navigation/types';
+import NAVIGATORS from '@src/NAVIGATORS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+
+// Moved to a separate file to avoid cyclic dependencies.
+const config: LinkingOptions<RootStackParamList>['config'] = {
```

Comment:
I'm finding this linking config really hard to reason + reason about. Any thoughts on splitting this up into multiple files to make it easier to understand how the linking config is composed together?